modules/simple-payments/simple-payments.php

```diff
@@ -157,7 +157,7 @@ function parse_shortcode( $attrs, $content = false ) {
 
 		if ( ! $this->is_enabled_jetpack_simple_payments() ) {
 			if ( ! is_feed() ) {
-				$this->output_admin_warning( $data );
+				return $this->output_admin_warning( $data );
```

Comment:
If the dependency change is an issue, then we could just make this change and deal with the dependency upgrade in another PR.

Reply:
Done :)